RAW/ViewController.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  ViewController.swift
+//  RAW
+//
+//  Created by Bharat malhotra on 13/07/18.
+//  Copyright © 2018 Bharat malhotra. All rights reserved.
+//
+
+import UIKit
+
+class ViewController: UIViewController {
+
+    @IBOutlet weak var tableView: UITableView!
+    var posts : [PostModel] = []
+    
```

Comment:
why are you assigning an empty array to the declaration?

Reply:
It should be like this [PostModel]()?